README.md

```diff
@@ -78,6 +79,8 @@ DATABASE_URL=postgresql://[username]:[password]@localhost:5432/[database]?schema
 2. Ask in `#team-projectlab` channel on _Slack_
    for the environment values for `.env`.
 
+For the `GOOGLE_APPLICATION_CREDENTIALS=` there will be a file named `wizelake-prod-wizelabs.json` in the files of the Slack group, you need to save it and use its route as your `GOOGLE_APPLICATION_CREDENTIALS=`. There's a example of it in `.env.example`.
```

Comment:
This should say in the keybase team, instead of the slack group

Reply:
oh yeah, mb